DEPLOYMENT.md

```diff
@@ -0,0 +1,7 @@
+# Manually deploying the contents of the build folder
+
+- run `yarn run build copy test`
+- run `firebase deploy --only=hosting`
```

Comment:
It is probably better to not rely on global `firebase` (if possible - I think it is).
I.e. do `yarn firebase deploy ...` instead.

Reply:
Hm, the cli tools are recommended to be installed globally: https://github.com/firebase/firebase-tools although it's probably safer to add them to package.json if it works. What are u doing in Angular? Aren't the docs using Firebase too?

Reply:
Yes, in Angular we use the local `firebase-tools` (afaict), but make it appear as if we use the global ones (which I believe is not intentional :smiley:):

https://github.com/angular/angular/blob/230b98d4ddffee4df25f62ff3c9ac39a2329abf6/aio/scripts/deploy-to-firebase.sh#L99-L100

The reason this works is because the `deploy-to-firebase.sh` script is invoked via a [yarn command][1], which means that local `node_modules/.bin/` is added to the path :grin:

[1]: https://github.com/angular/angular/blob/230b98d4ddffee4df25f62ff3c9ac39a2329abf6/aio/package.json#L37